Number_types/include/CGAL/Quotient.h

```diff
@@ -731,13 +732,13 @@ template < class NT > class Real_embeddable_traits_quotient_base< Quotient<NT> >
 
         std::pair< Interval_nt<>, int64_t > get_interval_exp( NT& x ) const {
 
-          CGAL_assertion(x >= zero);
+          CGAL_assertion(x >= 0);
           int64_t d = 0;
           double l = 0.0, u = 0.0;
           const int64_t n = static_cast<int64_t>(msb(x));
           const int64_t num_dbl_digits = std::numeric_limits<double>::digits - 1;
```

Comment:
I think that `-1` should be removed. The real value should be 53, and not 52.

Reply:
It fails with 53 but does not with 52, similar to the other implementation that we discussed together.

Reply:
I think it is related to the fact that `msb` is an index starting from 0 while `digits` is the size.

Reply:
Here in `get_interval_exp`, I understand, because `num_dbl_digits` is compared, or substracted to `msb(x)`. I would actually write add `+ 1` to the msb value, to avoid confusion.
```c++
          const int64_t n = static_cast<int64_t>(msb(x) + 1);
          const int64_t num_dbl_digits = std::numeric_limits<double>::digits;
```

Reply:
I see.

Reply:
I am now using `msb + 1` and I also tested that. It works.